.github/workflows/lint.yml

```diff
@@ -12,8 +12,9 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2.4.0
       - name: Run Super-Linter
-        uses: github/super-linter@v4.8.2
+        uses: github/super-linter@v4.8.3
         env:
+          FILTER_REGEX_EXCLUDE: .*docs/.*
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           VALIDATE_GO: false # Go linting is broken for now, see: https://github.com/github/super-linter/issues/143
```

Comment:
Does the issue really apply to us? It is related to linting isolated source files. Do we (need to) do that?

Reply:
Yes, linting isolated files breaks some linters. It simply doesn't work. I re-checked it recently and was still broken.